packages-web/maps/tests/e2e/pages/page.ts

```diff
@@ -0,0 +1,51 @@
+export class Page {
+    open(url = ""): void {
+        browser.url("/" + url);
+    }
+
+    getElement(name: string): WebdriverIO.Element {
+        const element = $(name);
+        element.waitForDisplayed();
+        return element;
+    }
+
+    getElements(name: string): WebdriverIO.Element[] {
+        $(name).waitForDisplayed();
+        const elements = $$(name);
```

Comment:
Maybe waitForDisplay for every elements:

`elements.forEach(element => element.waitForDisplayed());`

remove the first ` $(name).waitForDisplayed();`

Reply:
won't do. discussed